next.config.js

```diff
@@ -0,0 +1,40 @@
+const withLess = require('@zeit/next-less');
+const withCss = require('@zeit/next-css');
+const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin');
+
+module.exports = withCss(
```

Comment:
`nextCss` isn't necessary anymore as of next 9.2

Reply:
That is for pure CSS, and SCSS if i recall correctly.

It is still needed for LESS support: https://github.com/zeit/next-plugins/issues/598